kubernetes/charts/edge-kubernetes/templates/iotedged-pvc.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.iotedged.data.persistentVolumeClaim }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.iotedged.data.persistentVolumeClaim.name | quote }}
+  namespace: {{ include "edge-kubernetes.namespace" . | quote }}
+spec:
+  resources:
+    requests:
+      storage: 100Mi
```

Comment:
>100Mi [](start = 15, length = 5)

I wonder if we should expose this in values.yaml, too.  100M is way more that typical expected usage for HSM.